README.md

```diff
@@ -135,6 +135,11 @@ IngressDescriptor:
             name: public.language
 ```
 
+### `--ingress-descriptor` argument
```

Comment:
I think there could be a reference to the short version -i here

Reply:
> Apart from the doc, I've nothing to say, it looks good to me!

And the CHANGELOG :)